content/projects/t2f-pharm.md

```diff
@@ -0,0 +1,30 @@
+---
+title: T²F-Pharm
+date: 2020-03-03
+publishdate: 2020-03-03
+weight: 10
+nav: false
+people:
+- key: pratik.dhakal
+- key: jeremie.mortier
+- key: andrea.volkamer
+funding:
+- name: Bundesministerium für Bildung und Forschung, grant ID 031A262C
+publications:
+- t2f2018
+---
+
+Pharmacophore models are an accurate and minimal tridimensional abstraction of intermolecular interactions between chemical structures, usually derived from a group of molecules or from a ligand-target complex.
+Only a limited amount of solutions exists to model comprehensive pharmacophores using the information
+of a particular target structure without knowledge of any binding ligand.
+In the T²F-Pharm project, we developed an automated and customable tool for truly target-focused pharmacophore modeling.
+
+<!--more-->
```

Comment:
As I understand, this line indicates that the content below is only shown when we actually visit the project (here `t2f-pharm.md`), but not when visiting the parent site (here `pharmacophores.md`).

However, the full text is shown here
https://launch--admiring-blackwell-3e6756.netlify.app/research/structure-based/pharmacophores/

as opposed to e.g. here
https://launch--admiring-blackwell-3e6756.netlify.app/research/structure-based/binding-sites/

The behavior seems different to me, but maybe I am misled here.

Reply:
Thanks for the excellent catch, again! Yes, we were using `content` shortcodes instead of `intro`. Fixed!